cmd/main.go

```diff
@@ -61,8 +61,8 @@ func main() {
 		os.Exit(1)
 	}
 
-	additionalCASrc := fmt.Sprintf("%s/trusted-ca.crt", builder.ConfigMapCertsMountPath)
-	additionalCADst := fmt.Sprintf("%s/openshift-trusted-ca.crt", tlsCertRoot)
+	additionalCASrc := fmt.Sprintf("%s/additional-ca.crt", builder.ConfigMapCertsMountPath)
+	additionalCADst := fmt.Sprintf("%s/openshift-additional-ca.crt", tlsCertRoot)
```

Comment:
why are we changing the name between the source and destination?

Reply:
I kept the pattern with the service signing cert, though this probably is not necessary.